WordPress/src/main/java/org/wordpress/android/ui/utils/TitleSnackbarSpannable.kt

```diff
@@ -0,0 +1,42 @@
+package org.wordpress.android.ui.utils
+
+import android.content.Context
+import android.graphics.Color
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.ForegroundColorSpan
+import androidx.core.graphics.blue
+import androidx.core.graphics.green
+import androidx.core.graphics.red
+import org.wordpress.android.R
+import org.wordpress.android.util.extensions.getColorFromAttribute
+
+object TitleSnackbarSpannable {
```

Comment:
I was wondering if we should rename it to something else, considering it's used for the title **and** subtitle.  
`TitleSubtitleSnackbarSpannable`? `SnackbarTitleSubtitleSpannable`? `SnackbarTextSpannable`? `SnackbarContentSpannable`? I don't know 😂